### ISSUE TITLE
Makefile clean target doesn't work with GNU make

### ISSUE THREAD

**Issue by sten0** [rnkn/fountain-mode#137]
Hi Paul,

While importing the latest release into Debian, I noted the clean target was broken, but I'm assuming that you don't see it because you're on macOS and are using BSD make.

This is the problem:
https://github.com/rnkn/fountain-mode/blame/f3685ac7235de79cfa9a9c84acbbe8541bcf5e25/Makefile#L60

Please let me know if you'd like to support developers who use GNU (rather than BSD) systems, as well as if you'd prefer the  one-line solution that calls `find`, or something else.

**Comment by rnkn** [comment]
Thanks for the heads up. I've only considered the Makefile as a convenience thing for me, so portability hasn't been a huge concern, but if you have a fix, by all means!

**Comment by rnkn** [comment]
I just tested with GNU Make versions 3.81 (macOS) and 4.4.1 (gmake, MacPorts) and didn't get any errors, but if you could let me know what needs to change that would be great!

**Comment by sten0** [comment]
>
> I just tested with GNU Make versions 3.81 (macOS) and 4.4.1 (gmake,
> MacPorts) and didn't get any errors, but if you could let me know what
> needs to change that would be great!
>

Hm, well, it didn't look like shell expansion last I looked...oh!  I bet
macOS comes with a library of convenient macro expansions that are enabled
by default (I'm assuming this isn't a user config thing).

I've been waiting for your reply to learn if you'd like a PR that uses
native makefile (uglier, bigger, but works everywhere) or the conventional
method of calling 'find' (might not work on Windows).  Alternatively, if
you can ant to do it yourself, there are lots of resources online about
these two methods.

>

**Comment by rnkn** [comment]
Yes please send a patch if it's not working for you! I think the first way, the native Makefile would be best.
